packages/preview/isc-hei-report/0.1.5/isc_template_report.typ

```diff
@@ -0,0 +1,372 @@
+// Template for ISC reports at the School of engineering
+// v0.1.5 - mui 2024
+//
+// Missing features : 
+// - page and locations (above, under) references for figures not available yet
+
+// Fancy pretty print with line numbers and stuff
+#import "@preview/codelst:2.0.1": sourcecode 
+
+// Nice color boxes
+#import "@preview/showybox:2.0.1": showybox
+
+// Define our own functions
+#let todo(body, fill-color: yellow.lighten(50%)) = {
+  set text(black)
+  box(
+    baseline:25%,
+    fill: fill-color,
+    inset: 3pt,
+    [*TODO* #body],
+  )
+}
+
+//
+// Multiple languages support
+//
+
+// Thanks @LordBaryhobal for the original idea
+#let langs = (
+  fr: (
+    toc-title: "Table des matières",
+    bib-title: "Bibliographie",
+    chapter-title: "Chapitre",
+    figure-name: "Figure",
+    table-name: "Table",
+    listing-name: "Listing",    
+    appendix-title: "Annexes",
+    appendix-code-name: "Code annexé",
+    acronym-table-title: "Table des acronymes",
+    figure-table-title: "Table des figures",
+    listing-table-title: "Table des listings",    
+    table-table-title: "Table des tableaux",    
+  ),
+  en: (
+    toc-title: "Table of contents",
+    bib-title: "Bibliography",
+    chapter-title: "Chapter",
+    figure-name: "Figure",
+    table-name: "Table",
+    listing-name: "Listing",
+    appendix-code-name: "Code appendix",
+    appendix-title: "Appendix",
+    acronym-table-title: "Table of acronyms",
+    figure-table-title: "List of figures",
+    listing-table-title: "List of listings",
+    table-table-title: "List of tables",
+  ),
+  de: (
+    toc-title: "Inhaltsverzeichnis",
+    bib-title: "Literatur",
+    chapter-title: "Kapitel",
+    figure-name: "Abb.",    
+    table-name: "Tabelle",
+    listing-name: "Listing",
+    appendix-code-name: "Kode Anhang",
+    appendix-title: "Anhang",
+    acronym-table-title: "Akronymtabelle",
+    figure-table-title: "Abbildungen",
+    listing-table-title: "Quelltexte",
+    table-table-title: "Tabellen",
+  ),
+)
+
+#let i18n(lang, key) = {
+  if not lang in langs {
+    lang = "fr"
+  }
+  
+  let keys = langs.at(lang)
+
+  if not key in keys {
+    panic("I18n key " + key + "doesn't exist")
+  }
+  return keys.at(key)
+}
+
+
+//
+// Source code inclusion
+//
+#let luma_background = luma(250)
```

Comment:
If this value is public it would be better to use kebab-case to be consistent with the rest of the package. If you want to keep the snake_case name and it is not intended to be used outside of this file, maybe you can make that more explicit by following the underscore-prefix convention and renaming to `_luma_background`.

Reply:
This has been fixed in 23008a5
PDFs have been removed as well, sorry.

Reply:
No worries, thank you for taking time to fix it.